mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -363,7 +423,6 @@ def remove_repeated_time_index(ds):  # {{{
     return ds  # }}}
 
 
-
 def test_load_mpas_xarray_datasets(path):  # {{{
```

Comment:
We need to 1) update the tests at this point to be compatible with the changes above, or 2) we should remove the tests below and delegate to the py.test framework.  Either option is ok but I think the 2nd is cleaner.